packages/x-components/src/x-modules/facets/components/lists/selected-filters.vue

```diff
@@ -62,6 +65,20 @@
     @Getter('facets', 'selectedFiltersByFacet')
     public selectedFiltersByFacet!: FiltersByFacet;
 
+    /**
+     * Transform a dictionary of Facets including the slot name.
+     *
+     * @returns A dictionary of facets with the slot name.
+     *
+     * @internal
+     */
+    protected get mappedSelectedFilters(): Filter[] {
+      return this.selectedFilters.map((filter: any) => ({
+        ...filter,
+        slotName: toKebabCase(filter['facetId'] ?? '')
```

Comment:
Nice try casting filter to `any` type, but no one scapes the types police. As not every filter has to belong to a facet, you can check it with `isFacetFilter` function, and if it does not belong to a facet, use the `default` slot.

As this component is used by the `selected-filters-list` component, I won't complain about using a slightly different variation, but please, add the `slotName` property to an interface and use it as the return type here.